gradebookng/tool/src/java/org/sakaiproject/gradebookng/business/util/ImportGradesHelper.java

```diff
@@ -52,8 +52,8 @@
 	public final static int USER_NAME_POS = 1;
 
 	// patterns for detecting column headers and their types
-	final static Pattern ASSIGNMENT_WITH_POINTS_PATTERN = Pattern.compile("([\\w ]+ \\[[0-9]+(\\.[0-9][0-9]?)?\\])");
-	final static Pattern ASSIGNMENT_COMMENT_PATTERN = Pattern.compile("(\\* [\\w ]+)");
+	final static Pattern ASSIGNMENT_WITH_POINTS_PATTERN = Pattern.compile("(\.\* \\[[0-9]+(\\.[0-9][0-9]?)?\\])");
```

Comment:
In a regex you always need zero or 2 backslashes as you first have to get past the string escaping.

I don't think you need any escaping for a `.*` as you don't need to escape them in a a string.